lib/OpenQA/Worker/Job.pm

```diff
@@ -964,9 +964,11 @@ sub post_upload_progress_to_liveviewhandler {
     $self->{_progress_info} = \%new_progress_info;
 
     my $job_id = $self->id;
+    my $global_settings = OpenQA::Worker::Settings->new->global_settings;
```

Comment:
Even better, we don't have to retrieve the worker settings at all once the web UI is telling the worker what the service port config is.

Reply:
Depends on where the worker will store that port. Probably you'd add another attribute on worker level for this so the code accessing it would become `$self->worker->service_port_delta`.

Reply:
I've add it as a attribute pretty much exactly the same as the worker_id that is also received from the web Ui when it registers a worker.

Reply:
Patch is coming as soon as i think it has a chance of passing the CI tests

Reply:
About CI failures: The javascript style issues should be fixed when you rebase your branch onto current master where a fix was applied. The other test failures are certainly related, e.g.
* https://app.circleci.com/pipelines/github/os-autoinst/openQA/12812/workflows/ba5fce3d-e4ab-466f-a823-86ba92b79cd0/jobs/119431 stating "   #     message   => "http://localhost:9526/asset/6229f1fe02/bootstrap.js 269:246 \"jQuery.Deferred exception: Failed to construct 'WebSocket': The URL 'ws:/localhost:NaN/liveviewhandler/tests/1/developer/ws-proxy' is invalid.\" \"Error: Failed to construct 'WebSocket': The URL 'ws:/localhost:NaN/liveviewhandler/tests/1/developer/ws-proxy' is invalid.\\n    at establishWebSocketConnection (http://localhost:9526/asset/285cd7a70d/ws_console.js:9:48)\\n    at setupWebSocketConsole (http://localhost:9526/asset/285cd7a70d/ws_console.js:15:230)\\n    at HTMLDocument.\\u003Canonymous> (http://localhost:9526/tests/1/developer/ws-console?proxy=1:34:17)\\n    at mightThrow (http://localhost:9526/asset/6229f1fe02/bootstrap.js:259:18)\\n    at process (http://localhost:9526/asset/6229f1fe02/bootstrap.js:261:89)\" undefined",
* https://app.circleci.com/pipelines/github/os-autoinst/openQA/12812/workflows/ba5fce3d-e4ab-466f-a823-86ba92b79cd0/jobs/119428 with

```
    #   Failed test '"test" configuration'
    #   at t/config.t line 184.
    #     Structures begin differing at:
    #          $got->{global}{service_port_delta} = '2'
    #     $expected->{global}{service_port_delta} = Does not exist
```